src/object.d

```diff
@@ -1959,6 +1963,37 @@ void* aaLiteral(Key, Value)(Key[] keys, Value[] values) @trusted pure
 }
 
 alias AssociativeArray(Key, Value) = Value[Key];
+/*
+public template AssocArray(K, V)
+{
+    import assoc_array;
```

Comment:
Good to see the import being delayed here. I'd recommend to put assoc_array into some sub package of core or core.internal, though.

Reply:
I'd insist on that.